examples/forms.amp.html

```diff
@@ -756,3 +756,26 @@ <h1>Here are the results for the search: {{term}}</h1>
 
 </body>
 </html>
+
+<h4>Using a submit-success template referenced by id</h4>
+
+<template type="amp-mustache" id="submit_success_template">
+    <h1>You searched for: {{term}}</h1>
+</template>
+
+<form method="get"
+      action="https://httpbin.org/response-headers?AMP-Access-Control-Allow-Source-Origin=http%3A%2F%2Flocalhost%3A8000&access-control-expose-headers=AMP-Access-Control-Allow-Source-Origin"
+      action-xhr="https://httpbin.org/response-headers?AMP-Access-Control-Allow-Source-Origin=http%3A%2F%2Flocalhost%3A8000&access-control-expose-headers=AMP-Access-Control-Allow-Source-Origin"
+      target="_blank">
+    <fieldset>
+        <input name="clientId" type="hidden" value="CLIENT_ID(poll)" data-amp-replace="CLIENT_ID">
+        <input name="canonicalUrl" type="hidden" value="RANDOM and CANONICAL_URL" data-amp-replace="CANONICAL_URL RANDOM">
+        <label>
+            <span>Search for</span>
+            <input type="search" name="term" required>
+        </label>
+        <input type="submit" value="Search">
+    </fieldset>
+
+    <div submit-success template="submit_success_template"></div>
```

Comment:
Althought this works in runtime, it requires validator changes. Currently validator would complain and say `The attribute 'template' may not appear in tag 'FORM > DIV `

Reply:
@aghassemi I added this to `validator-main.protoascii`

@honeybadgerdontcare is there a way to create a validator rule to say "either `<div submit-success>` has an attribute `template` OR a child node `<template>` BUT not both"?

Reply:
Ping @honeybadgerdontcare